packages/storybook/stories/snippets/localization/AsyncLocales.snippet.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  LocalizationProvider,
+  COMPONENT_LOCALE_JA_JP,
+  COMPONENT_LOCALE_FR_FR,
+  COMPONENT_LOCALE_EN_US,
+  ComponentLocale,
+  CameraButton
+} from '@azure/communication-react';
+import React from 'react';
+
+export const AsyncLocalesSnippet = (): JSX.Element => {
+  //  mock API functions

```

Comment:
Could we hose an API service for our locales instead of exporting them?

Reply:
Given that we are purely a UI framework. I would discourage it.
Can we use something like azure blob storage directly without an API?

Reply:
I see. We can also move the locales to a separate package. It seems that 'runtime loading' is a bit of a misnomer and we don't actually need to load locales at runtime. It seems the need is for users to only include locales that they need.

Reply:
Customers may want to only use a certain number of locales but its the user that needed to choose a specific one which is done at runtime.

1. Lets make the locales available in some form like JSON. 
2. We can go one step further for our internal samples where we host it similar to our token service endpoints. 
3. We then document best practices if someone else wanted to use this JSON.
By doing it like this we have enabled customers to use a number of different localizations but they have to hook it up themselves without adding the burden onto our team for maintaining a locale providing service.